docs/architecture/adr-069-unordered-account.md

```diff
@@ -0,0 +1,106 @@
+# ADR 069: Un-Ordered Account
+
+## Changelog
+
+* Nov 24, 2023: Initial Draft
+
+## Abstract
+
+Support gaps in account nonce values to support un-ordered(concurrent) transaction inclusion.
+
+## Context
+
+Right now the transactions from a particular sender must be included in strict order because of the nonce value requirement, which makes it tricky to send many transactions concurrently, for example, when a previous pending transaction fails or timeouts, all the following transactions are all blocked. Relayer and exchanges are some typical examples.
+
+The main purpose of nonce value is to protect against replay attack, but for that purpose we only need to make sure the nonce is unique, so we can relax the orderness requirement without lose the uniqueness, in that way we can improve the user experience of concurrent transaction sending.
```

Comment:
```suggestion
Right now, transactions from a particular sender must be included in strict order
because of nonce value requirements, which makes it tricky to send many transactions
concurrently. For example, when a previous pending transaction fails or timeouts,
all the following transactions are blocked. Relayer and exchanges are some typical
examples.

The main purpose of nonce value is to protect against replay attack, but for that
purpose we only need to make sure the nonce is unique, so we can relax the order
requirement without loss to uniqueness, in that way we can improve the user experience
of concurrent transaction sending.
```